src/configurations/destinations/rs/schema.json

```diff
@@ -596,6 +580,66 @@
       }
     },
     "allOf": [
+      {
+        "if": {
+          "properties": {
+            "useIAMForAuth": {
+              "const": false
+            }
+          },
+          "required": ["useIAMForAuth"]
+        },
+        "then": {
+          "properties": {
+            "host": {
+              "type": "string",
+              "pattern": "(^\\{\\{.*\\|\\|(.*)\\}\\}$)|(^env[.].+)|(?!.*\\.ngrok\\.io)^(.{1,255})$"
```

Comment:
I understand this is copy/pasted, but why do we exclude ngrok.io from the host?

Reply:
Not sure. It's added by the integrations team. https://github.com/rudderlabs/rudder-integrations-config/pull/226
@ujjwal-ab Any idea why we want to disallow the `ngrok`.